packages/rpcs/wallet_enrichConfluxTx/index.js

```diff
@@ -39,8 +40,8 @@ export const main = async ({
   }
 
   if (to) {
-    const decoded = decode(to)
-    if (decoded.type !== 'contract')
+    decoded = decode(to)
+    if (decoded.type !== 'contract' || !data)
```

Comment:
|| data = 0x0 ?

Reply:
the data is already formatted in sign transaction